pkg/proxy/ipvs/netlink_linux.go

```diff
@@ -21,8 +21,10 @@ package ipvs
 
 import (
 	"fmt"
+	"net"
 
 	"k8s.io/apimachinery/pkg/util/sets"
+	utilproxy "k8s.io/kubernetes/pkg/proxy/util"
```

Comment:
this seems the reasons that verify check breaks, since it doesn't compile in windows

> ERROR(windows/arm64): /home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/pkg/proxy/ipvs/proxier_test.go:5429:9: undeclared name: netlinkHandle
> ERROR(windows/arm64): /home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/vishvananda/netlink/xfrm.go:13:36: IPPROTO_ROUTING not declared by package unix
> ERROR(windows/arm64): /home/prow/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/vishvananda/netlink/xfrm.go:14:36: IPPROTO_ESP not declared by package unix